README.md

```diff
@@ -135,6 +136,7 @@ See also https://github.com/timotheecour/D_vs_nim/issues/11
 | increment | i++ | i+=1 or inc(i) |
 | concatenation | ~ | & |
 | empty statement | {} | discard |
```

Comment:
maybe add a line here (i know u've mentioned it earlier but it's lost in text) showing `a is null` vs `a == nil`

Reply:
Added a “checking for null” row right under the “null pointer” row